### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by aamiel1** [eusonlito/GPS-Tracker#53] [Ideas]
Do you going to develop an API to the system?

**Comment by eusonlito** [comment]
It is my intention to add API endpoints in the future, but I currently do not have enough time for implementation.

What do you think should be the functionalities to start adding API endpoints?

**Comment by aamiel1** [comment]
I think is to start with:
* Login
* Create Car
* Create Device

**Comment by eusonlito** [comment]
Hi @aamiel1, I have added some API endpoints.

Wiki: https://github.com/eusonlito/GPS-Tracker/wiki/%5BEN%5D-API
